lib/engine/game/g_18_eus/step/loan_taker.rb

```diff
@@ -43,6 +43,19 @@ def process_payoff_loan(action)
             @round.paid_loans |= [action.entity]
             pass!
           end
+
+          def game_buttons(entity)
+            buttons = super
+            if can_take_loan?(entity)
+              action = Engine::Action::TakeLoan.new(entity, loan: nil)
```

Comment:
Is `loan: nil` needed? Does it not default to that?

Reply:
It does not.